### PR TITLE
Remove django ORM from manual_remove.py

### PR DESCRIPTION
**Pull request by ElliotAOram**
### Summary of work
* Remove use of old database SQLAlchemy connections in manual_remove.
* Update tests to use Django ORM 

### How to test your work
* Ensure the tests pass
* Attempt to use the script to manually remove a run from the database
   * see instructions in `scripts/manual_operations/README.md` 

Part of #546 

**Before merging ensure the release notes have been updated**

### CODE REVIEW

**Comment by ElliotAOram** [comment]
will need to re-add test cases for database interactions

**Comment by ElliotAOram** [comment]
This is now ready to review, the drop in test coverage is because I have removed obsolete code that was previously tested (coveralls always gets a bit annoyed at this)

File: scripts/manual_operations/manual_remove.py [hunk 1]
```diff
@@ -10,37 +10,31 @@
...
     """
     Handles removing a run from the database
     """
 
-    def __init__(self, instrument, credentials=None):
-        if credentials:
-            self.database = DatabaseClient(credentials)
-        else:
-            self.database = DatabaseClient()
+    def __init__(self, instrument):
```
Comment (JackEAllen) [inline scripts/manual_operations/manual_remove.py:21]: NIT: Should we consider adding a docstring with parameters here?

File: scripts/manual_operations/manual_remove.py [hunk 2]
```diff
@@ -10,37 +10,31 @@
...
+        self.database = DatabaseClient()
         self.database.connect()
         self.to_delete = {}
         self.instrument = instrument
 
     def find_runs_in_database(self, run_number):
         """
-        Find all runs in the database that relate to a given instrument and run number
+        Find all run versions in the database that relate to a given instrument and run number
         :param run_number: the run to search for in the database
```
Comment (JackEAllen) [inline scripts/manual_operations/manual_remove.py:30]: Would it be worth adding the datatype to the parameters and return values in docstrings to be more explicit with method usage? 
`:param run_number: (int) the run to search for in the database`

File: scripts/manual_operations/manual_remove.py [hunk 3]
```diff
@@ -101,52 +95,60 @@ def delete_records(self):
...
-                                   self.database.reduction_run().id == version.id)
+                self.delete_reduction_run(version.id)
 
             # Remove deleted run from dictionary
             del self.to_delete[run_number]
 
+    def delete_reduction_location(self, reduction_run_id):
+        """
+        Delete a ReductionLocation record from the database
+        :param reduction_run_id: The id of the associated reduction job
```
Comment (JackEAllen) [inline scripts/manual_operations/manual_remove.py:109]: NIT: Again here with adding datatype to docstring

File: scripts/manual_operations/manual_remove.py [hunk 4]
```diff
@@ -101,52 +95,60 @@ def delete_records(self):
...
+        """
+        self.database.data_model.ReductionLocation.objects \
+            .filter(reduction_run_id=reduction_run_id) \
+            .delete()
+
+    def delete_data_location(self, reduction_run_id):
+        """
+        Delete a DataLocation record from the database
+        :param reduction_run_id: The id of the associated reduction job
```
Comment (JackEAllen) [inline scripts/manual_operations/manual_remove.py:118]: NIT: Add datatype to docstring

File: scripts/manual_operations/manual_remove.py [hunk 5]
```diff
@@ -101,52 +95,60 @@ def delete_records(self):
...
+        self.database.data_model.DataLocation.objects \
+            .filter(reduction_run_id=reduction_run_id) \
+            .delete()
+
     def delete_variables(self, reduction_run_id):
         """
-        Removes all the run parameters associated
-        :return:
+        Removes all the RunVariable records associated with a given ReductionRun from the database
+        :param reduction_run_id: The id of the associated reduction job
```
Comment (JackEAllen) [inline scripts/manual_operations/manual_remove.py:127]: NIT: Add datatype to docstring

File: scripts/manual_operations/manual_remove.py [hunk 6]
```diff
@@ -101,52 +95,60 @@ def delete_records(self):
...
-                               record.variable_ptr_id)
+            self.database.variable_model.RunVariable.objects \
+                .filter(variable_ptr_id=record.variable_ptr_id) \
+                .delete()
+
+    def find_variables_of_reduction(self, reduction_run_id):
+        """
+        Find all the RunVariable records in the database associated with a reduction job
+        :param reduction_run_id: The id of the reduction job to filter by
+        :return: A QuerySet of the associated RunVariables
```
Comment (JackEAllen) [inline scripts/manual_operations/manual_remove.py:139]: NIT: Add datatype to docstring

File: scripts/manual_operations/manual_remove.py [hunk 7]
```diff
@@ -101,52 +95,60 @@ def delete_records(self):
...
+            .filter(reduction_run_id=reduction_run_id)
 
-    def delete_record(self, query_value, filter_on):
+    def delete_reduction_run(self, reduction_run_id):
         """
-        Run a delete on the database
-        :param query_value: the query to use
-        :param filter_on: filtering options
+        Delete a ReductionRun record from the database
+        :param reduction_run_id: The id of the associated reduction job
```
Comment (JackEAllen) [inline scripts/manual_operations/manual_remove.py:147]: NIT: Add datatype to docstring

**Comment by ElliotAOram** [comment]
@JackEAllen this is ready for review again

**Pull request merged**